dhctl/pkg/operations/bootstrap/state.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 Flant JSC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// TODO move all states from operations/bootstrap to here
+
+package bootstrap
+
+import "github.com/deckhouse/deckhouse/dhctl/pkg/state"
+
+const PostBootstrapResultCacheKey = "post-bootstrap-result"
```

Comment:
I completely missed the part about caching bootstrap results. Could you please explain to me why do we need to cache it? 

(From my point of view, users can write any script and output whenever they want. It should not be idempotent or reproducible).

Reply:
We can use result from script in another applications. Share information between another applications via cache (state).

Reply:
Let's keep things simple. I do not see cases when we need to share specific information through cache. From my point of view, it is a complication without benefits.

Reply:
Load balancer IP, for example

Reply:
We discussed it internally and agreed to implement this, yet slightly change the logic of how we parse the output.

```bash
#! /bin/bash

# $OUTPUT="/tmp/dhctl-exec-script-adasdasda"
kubectl get pods -A > $OUTPUT
```

* $OUTPUT is the env variable that points to the tmp file.
* At the end of the script, we read the data from this file and save it to the cache.